tier4_planning_msgs/msg/SafetyFactorArray.msg

```diff
@@ -0,0 +1,4 @@
+std_msgs/Header header
+tier4_planning_msgs/SafetyFactor[] factors
+bool is_safe
```

Comment:
Is boolean enough instead of state? Considering the operator cooperation feature, it seems better to express a more complex state.

Reply:
:thinking: